.gitpod.yml

```diff
@@ -0,0 +1,10 @@
+tasks:
+  - name: Run demo
+    init: pip install -r requirements.txt
+    command: |
+      python demo.py
```

Comment:
I decided that I don't think we should run this demo without a prompt from the user. I don't want someone with limited Leap time to open it in the IDE just to look at it and have it run when they don't want it to.

Reply:
Makes sense. I'll adapt this PR accordingly.